ahnlich/ai/src/server/handler.rs

```diff
@@ -90,7 +95,7 @@ pub struct AIProxyServer {
     client_handler: Arc<ClientHandler>,
     store_handler: Arc<AIStoreHandler>,
     task_manager: Arc<TaskManager>,
-    db_client: Arc<DbClient>,
+    db_client: Option<Arc<DbClient>>,
```

Comment:
So now we need a CLI argument such as `--without-db` with which to spin AI proxy up with for which db_client would be `None`.... the default should remain as is i.e it seeks to initialize a connection to DB with `--db-host` and `--db-port` ... both argument groups should be mutually exclusive i.e `--without-db` and `--db-*` arguments cannot be passed together perhaps with something like [this](https://rust.code-maven.com/clap/clap-mutually-exclusive-with-group)